### ISSUE TITLE
Determine whether FEAF taxonomy is sufficient for attributing to websites

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/EDX#374]
Determine whether the Business Reference Model (BRM) from the Federal Enterprise Architecture Framework (FEAF) is an effective taxonomy to use to attribute to websites. This test-case is being carried out in order to prevent us from having to develop a custom taxonomy from scratch. 

The ultimate goal of attributing a taxonomy to the websites is to be able to effectively compare the websites based on similar missions/services in order to determine redundancies or gaps and to aid in the governance process when approving/denying new public DNS requests.

**Comment by AKM399** [comment]
possibly look at shared traffic and/or performance within each group to see how comparable sites with similar business capabilities are (note to self) @AKM399

**Comment by hatfieldjm4** [comment]
In my analysis, I've determined that the FEAF taxonomy is sufficient for attributing to websites that also function as business applications (i.e. transactional applications that are used to accomplish a business need as opposed to information-only websites). For websites that simply display information and do not have a business functionality outside of displaying content on a page, there are only 1-2 FEAF business capabilities that align (most notably [1.01.08.7.1 Government Information Dissemination](https://ea.gsa.gov/#!/capabilities/200)). For the content-only sites (which accounts for about 50% of the public facing inventory, including login pages), I determined that creating keywords/topics for each site was a better way of categorizing them into themes that could be compared across the inventory. 

For consistency, I applied these keywords/topics to all sites (even the ones that would be considered business applications) so that they can be easily compared across the entire public-facing website inventory.  My analysis can be found [here](https://docs.google.com/spreadsheets/d/1FdX7Jtz8I96R87TnTlkOgs5plXMuq2Z9-bo0Vx2HIOU/edit#gid=431930911).
